Source/Core/Core/IOS/Network/KD/VFF/VFFUtil.cpp

```diff
@@ -21,45 +17,16 @@
 
 #include "Common/Align.h"
 #include "Common/FatFsUtil.h"
-#include "Common/FileUtil.h"
-#include "Common/IOFile.h"
 #include "Common/Logging/Log.h"
 #include "Common/ScopeGuard.h"
-#include "Common/StringUtil.h"
 #include "Common/Swap.h"
 
 #include "Core/IOS/Uids.h"
 
-static IOS::HLE::FS::FileHandle* s_vff;
-
-static DRESULT vff_read(BYTE pdrv, BYTE* buff, LBA_t sector, UINT count);
-static DRESULT vff_write(BYTE pdrv, const BYTE* buff, LBA_t sector, UINT count);
-static DRESULT vff_ioctl(BYTE pdrv, BYTE cmd, void* buff);
-static DRESULT read_vff_header(FATFS* fs);
-
-enum : u16
-{
-  SECTOR_SIZE = 512,
-  VF_LITTLE_ENDIAN = 0xFFFE,
-  VF_BIG_ENDIAN = 0xFEFF
-};
-
-static FRESULT vff_mount(FATFS* fs)
+DRESULT read_vff_header(IOS::HLE::FS::FileHandle* vff, FATFS* fs)
```

Comment:
This should be either `static` or in an anonymous namespace, since you don't want this symbol to be exported.